src/tracing/CMakeLists.txt

```diff
@@ -43,9 +43,9 @@ endfunction()
 
 set(osd_traces oprequest.tp osd.tp pg.tp)
 add_tracing_library(osd_tp "${osd_traces}" 1.0.0)
-add_tracing_library(rados_tp librados.tp 2.0.0)
+add_tracing_library(librados_tp librados.tp 2.0.0)
```

Comment:
i don't understand why it makes different if we use librados_tp as the tracing point library name for librados. and please note, the generated library would be named as "liblibrados_tp.so" if we go this way.

also, "rados-tp" is not "rados_tp". the former is the tracing library, the later is just a target for building the header file for librados's tracing points.